package.json

```diff
@@ -12,38 +12,38 @@
   },
   "private": true,
   "dependencies": {
-    "@angular/animations": "^12.0.5",
-    "@angular/cdk": "^12.0.5",
-    "@angular/common": "^12.0.5",
-    "@angular/compiler": "^12.0.5",
-    "@angular/core": "^12.0.5",
-    "@angular/flex-layout": "^12.0.0-beta.34",
-    "@angular/forms": "^12.0.5",
-    "@angular/material": "^12.0.5",
-    "@angular/platform-browser": "^12.0.5",
-    "@angular/platform-browser-dynamic": "^12.0.5",
-    "@angular/router": "^12.0.5",
-    "@jsonforms/angular": "3.1.0",
-    "@jsonforms/angular-material": "3.1.0",
-    "@jsonforms/core": "3.1.0",
+    "@angular/animations": "^17.0.8",
+    "@angular/cdk": "^16.0.0",
+    "@angular/common": "^17.0.8",
+    "@angular/compiler": "^17.0.8",
+    "@angular/core": "^17.0.8",
+    "@angular/flex-layout": "^15.0.0-beta.42",
```

Comment:
`@angular/flex-layout` should not be needed anymore. Thus the dependency can be removed